### PR TITLE
Landing page tweaks

### PR DESCRIPTION
**Pull request by Uberi**
Based on #102.

* Make the list of external dashboards a single scrollable list.
* Move the CSS out into a separate file.
* Graduate V4 dashboards from draft status (they are as stable as the v2 dashboards now).

### CODE REVIEW

**Comment by vdjeric** [comment]
Where can I test these changes? Does divshot have both #103 + #102 ?

**Comment by Uberi** [comment]
Yes, http://staging.telemetry-dashboard.divshot.io/ has the new landing page on top of #102, so it should be good for testing the listed changes.

**Comment by vdjeric** [comment]
I like how "Other dashboards" looks now.
"View measure value distributions as a histogram." is hard to parse, change to "View distributions as a histogram".
In a separate pull request, update the documentation for the V2 dash (screenshots, text, new functionality, etc)

**Comment by vdjeric** [comment]
Broken image link on http://staging.telemetry-dashboard.divshot.io/new-pipeline/tutorial.html

**Comment by Uberi** [comment]
* Rebased to latest new-pipeline.
    * Export links and tutorial images fixed.
    * Dashboard description wording updated.
* Deployed to http://staging.telemetry-dashboard.divshot.io/

**Comment by vdjeric** [comment]
Bug fixes:
* Date selector doesn't match legend: http://bit.ly/1hwfo4g
* Why is there overlap between these lines? http://bit.ly/1hwfDML

Enhancements:
* "compare by" should not show a 3 point curve for boolean histograms, just use a 2-point line for now
